app/src/test/groovy/gov/hhs/cdc/trustedintermediary/external/in_memory/keyCacheTest.groovy

```diff
@@ -0,0 +1,46 @@
+package gov.hhs.cdc.trustedintermediary.external.in_memory
+
+import gov.hhs.cdc.trustedintermediary.external.in_memory.KeyCache
+import spock.lang.Specification
+
+class keyCacheTest extends Specification {
```

Comment:
Rename this file (and this class) to have a beginning capital letter.

Reply:
Good catch!